test/Microsoft.ML.Functional.Tests/Datasets/ScoreColumn.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.ML.Functional.Tests.Datasets
+{
+    /// <summary>
+    /// A class to hold a feature column.
+    /// </summary>
+    internal sealed class ScoreColumn
+    {
+        public float Score { get; set; }
+    }
```

Comment:
can this go somewhere, not on a separate file?

Reply:
Yes. I've consolidated all the single-column classes into one file. I had been following the Apache REEF style by habit.

---
In reply to: [265712606](https://github.com/dotnet/machinelearning/pull/2899#discussion_r265712606) [](ancestors = 265712606)